tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/ConfidentialClientIntegrationTests.cs

```diff
@@ -162,14 +181,16 @@ public async Task ConfidentialClientWithRSACertificateTestAsync()
         }
 
         [TestMethod]
-        public async Task ConfidentialClientWithClientSecretTestAsync()
+        [DataRow(ConfidentialClientID, TestAuthority, "default", DisplayName ="Public Cloud")]
+        [DataRow(_arlingtonConfidentialClientIDOBO, _arlingtonAuthority, "arlington", DisplayName ="Arlington Cloud")]
```

Comment:
DataRow is cool, but it should be used consistently - also in the ROPC and other tests.